lib/snapshot/test_command_generator.rb

```diff
@@ -70,7 +70,7 @@ def destination(device)
 
         device_udid = nil
         FastlaneCore::Simulator.all.each do |sim|
-          device_udid = sim.udid if sim.name.strip == device.strip
+          device_udid = sim.udid if sim.name.strip == device.strip and sim.ios_version.strip == Snapshot.config[:ios_version]
```

Comment:
no, you're quite right: we don't need to strip the sim.ios_version. Has the Snapshot config setting already been stripped?